model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py

```diff
@@ -35,9 +28,7 @@ def update(self, **kwargs):
         export_path=export_path,
         force=False,
         archive_format="default",
-        convert=False,
         version=version,
-        source_vocab=source_vocab,
```

Comment:
neither `convert` nor `source_vocab` are unsed in model archiver. Not [here](https://github.com/pytorch/serve/blob/master/model-archiver/model_archiver/model_packaging_utils.py#L110) anyway, could this break a behaviour?

Reply:
Let's keep them for now, this might be breaking something we're not testing

Reply:
I think its okay to remove them. They are not part of the namespace created in argparse and only show up in this test. Presumably a relicts from before extra_files.
```
(serve) ubuntu@ip-172-31-55-226:~/serve$ grep -r convert model-archiver/
model-archiver/model_archiver/model_packaging_utils.py:            logging.error("Failed to convert %s to the model-archive.", model_name)
Binary file model-archiver/model_archiver/tests/unit_tests/__pycache__/test_model_packaging.cpython-310-pytest-7.3.1.pyc matches
model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py:        convert=False,
model-archiver/build/lib/model_archiver/model_packaging_utils.py:            logging.error("Failed to convert %s to the model-archive.", model_name)
(serve) ubuntu@ip-172-31-55-226:~/serve$ grep -r source_vocab model-archiver/
Binary file model-archiver/model_archiver/tests/integ_tests/__pycache__/test_integration_model_archiver.cpython-310-pytest-7.3.1.pyc matches
model-archiver/model_archiver/tests/integ_tests/test_integration_model_archiver.py:        assert os.path.join(prefix, "source_vocab.pt") in file_list
model-archiver/model_archiver/tests/integ_tests/default_handler_configuration.json:    "extra-files": "model_archiver/tests/integ_tests/resources/regular_model/test_index_to_name.json,model_archiver/tests/integ_tests/resources/regular_model/source_vocab.pt",
Binary file model-archiver/model_archiver/tests/unit_tests/__pycache__/test_model_packaging.cpython-310-pytest-7.3.1.pyc matches
model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py:    source_vocab = None
model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py:        source_vocab=source_vocab,
```